custom_model_runner/datarobot_drum/drum/model_adapter.py

```diff
@@ -357,7 +357,10 @@ def _drum_autofit_internal(self, X, y, output_dir):
                         break
 
         if marked_object is not None:
-            marked_object.fit(X, y)
+            if y is not None:
+                marked_object.fit(X, y)
+            else:
+                marked_object.fit(X)
```

Comment:
do you think we should pass y=none here explicitly?

Reply:
yeah I guess there's no reason not to